depthai_sdk/docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Afaik not needed

Reply:
It was generated automatically so I didn't look into it 😄 will remove